packages/base-driver/lib/basedriver/helpers.js

```diff
@@ -546,6 +546,22 @@ function parseCapsArray (cap) {
   throw new Error(`must provide a string or JSON Array; received ${cap}`);
 }
 
+/**
+ * Generate a string that uniquely describes driver instance
+ *
+ * @param {Object} obj driver instance
+ * @param {string?} sessionId session identifier (if exists)
+ * @returns {string}
+ */
+function generateDriverLogPrefix (obj, sessionId = null) {
+  const instanceName = `${obj.constructor.name}@${node.getObjectId(obj).substring(0, 4)}`;
+  return sessionId ? `${instanceName} (${sessionId.substring(0, 8)})` : instanceName;
+}
+
 /** @type {import('@appium/types').DriverHelpers} */
-export default {configureApp, isPackageOrBundle, duplicateKeys, parseCapsArray};
-export {configureApp, isPackageOrBundle, duplicateKeys, parseCapsArray};
+export default {
```

Comment:
We should probably move these "double" exports across the codebase; they are not needed anywhere except maybe on the public API edges.  (just saying)

Reply:
it was like that before, so it does not make sense to me to change it in this PR